### DISCUSSION TITLE
卸载 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1022] [Announcements]
# 卸载 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/uninstall/

**Comment by zgfh** [comment]
如何卸载商业版？

**Comment by windsonsea** [comment]
商业版一般需要铲掉整个环境
